js/client.envirophat.ocf.js

```diff
@@ -0,0 +1,140 @@
+/** Copyright 2017 Vprime
```

Comment:
You shouldn't need this file anymore.

Reply:
that's right! it should not be in the js folder... I even noticed it being deleted in the git status message before I amended my commit...